src/pages/lessons/fundamentals/eth-intro-part-2.mdx

```diff
@@ -0,0 +1,279 @@
+>
+<LessonHeader title="A Developer's Guide to Ethereum, Pt. 2" />
+
+Welcome back! In the first installment of this series we covered a lot of ground
+while interacting with a simulated Ethereum network. At this point, you should
+have at least a fuzzy idea of how to answer these questions:
+
+- What's a blockchain and what’s in a block?
+- What makes Ethereum decentralized?
+- What is ether and why is it a necessary component of the network?
+
+In this lesson, we’ll build on these concepts and explore some of the implications
+for developers, so return to Part 1 if you skipped it or need a refresher.
+
+## What’s next? 
+
+We’re going to take a deeper look at how you can interact with the Ethereum network, starting with accounts. There are some significant differences between Ethereum accounts and Web 2.0 accounts.
+
+<Callout emoji="💡" size="md" variant="info">
+**Note**: “Web 2.0” was coined to describe the Internet era that introduced user-generated content, e.g., social media and blogs. Ethereum and other decentralized technologies are said to be a part of the next generation of the Internet: **Web 3.0**. The abbreviation, **Web3**, is used by libraries like web3.js and web3.py, and elsewhere around the ecosystem.
+</Callout>
+
+## Web2 vs. Web3
+
+It’s hard not to collect accounts in today’s web. You have one for each social media app, news site, delivery service, retailer, and transit provider — to name a few. Each of these accounts lives on the company's servers, which makes you subject to their terms and conditions, privacy policy, and security practices. Your account can be frozen, deleted, censored, or altered at the discretion of the host company.
+
+Web3 represents a paradigm shift for account management: you and you alone own your Ethereum accounts. When you create an account, it’s done outside of the scope of any company and can travel with you to multiple apps. In fact, creating an Ethereum account doesn’t require interaction with the Ethereum blockchain at all. Let’s create one now to prove it.
+
+<Callout emoji="💡" size="md" variant="info">
+**Note**: This exercise is purely for educational purposes. Don't store real value in an account unless you understand the security implications. Some mistakes cannot be undone! More context to come.
+</Callout>
+
+## Creating an account
+
+Same drill as last time: the concepts will be demonstrated in an IPython shell. If you’re not a Python developer, no problem. Just follow along conceptually.
+
+### Environment setup
+
+Three steps to set the stage:
+
+1. Install web3.py, eth-tester, and IPython (if you didn't already in Part 1):
+
+```bash
+$ pip install web3 "web3[tester]" ipython
+```
+
+2. Start up a new sandbox:
+
+```bash
+$ ipython
+```
+
+3. Import the Web3 module:
+
+```python
+In [1]: from web3 import Web3
+```
+
+### Account generation
+
+Let's create that account:
+
+```python
+In [2]: w3 = Web3() # No provider necessary for this demo
```

Comment:
I believe we need to make it explicit to paste in the `# No provider necessary for this demo` other wise we 'do' end up with an output. 

I got the following, and went on to correct it: 
`In [1]: from web3 import Web3

In [2]: Web3()
Out[2]: <web3.main.Web3 at 0x7fd1d1d24c10>

In [3]: acct = w3.eth.account.create()
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
Cell In[3], line 1
----> 1 acct = w3.eth.account.create()

NameError: name 'w3' is not defined

In [4]: acct.address
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
Cell In[4], line 1
----> 1 acct.address

NameError: name 'acct' is not defined

In [5]: w3 = Web3() # No provider necessary for this demo

In [6]: acct = w3.eth.account.create()

In [7]: acct.address
Out[7]: '0xD8Bf88bd36D253167c5f129faAa1D09De91671f2'

In [8]: acct.key
Out[8]: HexBytes('0x94c789f4003025f345f3eecd519b84425eb654a1b5f731dbc465c22e38a98dd5')

In [9]: 
``